CUDADataFormats/EcalRecHitSoA/interface/EcalRecHit_soa.h

```diff
@@ -23,7 +23,8 @@ namespace ecal {
 
     typename type_wrapper<reco::StorageScalarType, L>::type energy;
     typename type_wrapper<reco::StorageScalarType, L>::type time;
-    typename type_wrapper<reco::StorageScalarType, L>::type chi2;  // should we remove this, since already included in "extra" ?
+    typename type_wrapper<reco::StorageScalarType, L>::type
+        chi2;                                           // should we remove this, since already included in "extra" ?
```

Comment:
I understand that the formatting is automatic - but this change is quite atrocious :-(

Reply:
I don't disagree. How about moving the comment to preceding line?

Reply:
Moved the comment.